theme_kiddo/views/snippets/s_product_list.xml

```diff
@@ -4,27 +4,29 @@
 <template id="s_product_list" inherit_id="website.s_product_list">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt96 pb96" separator=" "/>
+        <attribute name="class" add="pt96 pb96" remove="pt64 pb64" separator=" "/>
     </xpath>
 
     <!-- Container -->
-    <xpath expr="//div[hasclass('container-fluid')]" position="attributes">
-        <attribute name="class" add="container" remove="container-fluid" separator=" "/>
-    </xpath>
     <xpath expr="//div[hasclass('row')]" position="attributes">
         <attribute name="class" add="justify-content-center" separator=" "/>
     </xpath>
+
+    <!-- Title -->
+    <xpath expr="//h2" position="attributes">
+        <attribute name="style">text-align: center;</attribute>
+    </xpath>
     
     <!-- Item n°1 -->
     <xpath expr="//div[hasclass('col-lg-2')]" position="attributes">
         <attribute name="class" add="border rounded mx-3" separator=" "/>
         <attribute name="style">border-radius: 16px !important; border-width: 4px !important; border-style: dotted !important; border-color: var(--o-color-2) !important;</attribute>
     </xpath>
```

Comment:
Given that it is a card column, the border and rounding options are retargeted to the card, so you should remove them from the column and add it in the `s_card` instead :wink: 

If you find the border too close to the image, you can then add some padding on it (like how the image "Padding" option would do) :+1: 

You can also completely remove the `mx-3` class: 
- It prevents using the `offset-lg-*`  class, since they both set margins
- It won't be necessary anymore, since the border will be on the card, so they will be spaced thanks to the columns padding :slightly_smiling_face: